### PR TITLE
Fallback to fixed default if hessian cannot be computed

### PR DESCRIPTION
**Pull request by ingmarschuster**
This solves the case where the hessian cannot be computed. One way to do it would be to call the `approx_hessian()` function in that case. This broke down for me in 3000 dimensions, so I used a fixed hessian instead.
The `fixed_hessian()` function might be done more elegantly, unsure. Also, we might want to call `approx_hessian()` up to a certain dimension, `fixed_hessian()` for higher dimensions.
Another possibility would be to estimate the covariance from burn-in samples.

### CODE REVIEW

**Comment by fonnesbeck** [comment]
This looks fine to me.

File: pymc3/tuning/scaling.py [hunk 1]
```diff
@@ -78,7 +105,10 @@ def find_hessian_diag(point, vars=None, model=None):
 
 def guess_scaling(point, vars=None, model=None):
     model = modelcontext(model)
-    h = find_hessian_diag(point, vars, model=model)
+    try:
+        h = find_hessian_diag(point, vars, model=model)
+    except NotImplementedError:
```
Comment (twiecki) [inline pymc3/tuning/scaling.py:110]: When does this get raised?

Comment (ingmarschuster) [inline pymc3/tuning/scaling.py:110]: When you are using an individual Theano expression as the target distribution as in #804 and Theano can't automatically compute the second derivative of that expression. As for example when coding up the Stochastic Volatility target from the NUTS paper - it allows a first, but not a second derivative in Theano.

Comment (ingmarschuster) [inline pymc3/tuning/scaling.py:110]: If this pull request gets merged, I will open another one for #804.

**Comment by twiecki** [comment]
Thanks @ingmarschuster!

**Pull request merged**